drivers/ThinQ_Fridge.groovy

```diff
@@ -19,7 +19,9 @@ metadata {
 
         attribute "fridgeTemp", "number"
         attribute "freezerTemp", "number"
-        attribute "craftIceMode", "number"
+        attribute "fridgeTemp", "number"
+        attribute "freezerTemp°", "string"
+        attribute "craftIceMode°", "string"
```

Comment:
This doesn't seem right. craftIceMode is how many ice balls it makes per day which is a value of 3 or 6, not anything using degrees.

Reply:
Agreed, this got changed **accidentally** in my code duplication of the freezer/fridge attributes, good catch....